src/main/java/nl/tudelft/ti2206/group9/gui/MainMenuScene.java

```diff
@@ -98,4 +110,52 @@ public void handle(final ActionEvent event) {
 		});
 	}
 
+	/**
+	 * Checks whether the playername is a valid name.
+	 * Invalid options:
+	 * <a>
```

Comment:
Wasn't this the tag for a link? O:) Maybe you meant `<ul>`? :P

<documentation,review